### ISSUE TITLE
rgba regex errors when the alpha channel is 1.0

### ISSUE THREAD

**Issue by mlwohls** [pbugnion/gmaps#146]
I think rgba should allow a max of 1.0 but the regex is throwing errors if the alpha doesn't start with a 0.x

error message:
```Element of the 'gradient' trait of a WeightedHeatmap instance must be an HTML color recognized by Google maps or a tuple or a tuple, but a value of 'rgba(72,209,204,1.0)' <type 'str'> was specified.```

I think the error is being created by line 62 in geotraitlets.py:
`_rgba_re = re.compile(r'rgba\([0-9]{1,3},[0-9]{1,3},[0-9]{1,3},0?\.[0-9]*\)')`
the issue is the leading zero in `0?\.[0-9]*\)`

**Comment by pbugnion** [comment]
Good spot! Thanks for raising this.

Would you like to submit a PR?

**Comment by pbugnion** [comment]
Note that, for anyone looking for a work-around, you can always specify the color as a Python tuple. These do allow an alpha value of 1.0. E.g. (borrowing code from the tutorial):

```python
color = (100, 0, 250, 0.5)

gmaps.symbol_layer(starbucks_df, fill_color=color, stroke_color=color)
```

**Comment by mlwohls** [comment]
pbugnion - I was actually passing in Python tuples when it was throwing the error. The 'rgba(72,209,204,1.0)' string was what the script converted the Tuple to, then errorer out in the final validation I guess.

**Comment by mlwohls** [comment]
I'll see if I can hack up a better RegEx.  My RegEx sucks, but I'll see if I can figure it out.

**Comment by mlwohls** [comment]
PR submitted.
https://github.com/pbugnion/gmaps/pull/147

**Comment by pbugnion** [comment]
> pbugnion - I was actually passing in Python tuples when it was throwing the error

Ah -- my bad. I'd forgotten how this part of the code works.

> PR submitted.

Thanks a lot! I'll review it tomorrow.

**Comment by pbugnion** [comment]
Now fixed by PR #150.
